pilot/pkg/xds/ads.go

```diff
@@ -575,24 +564,24 @@ func (s *DiscoveryServer) DeltaAggregatedResources(stream discovery.AggregatedDi
 // for large configs. The method will hold a lock on con.pushMutex.
 func (s *DiscoveryServer) pushConnection(con *Connection, pushEv *Event) error {
 	// TODO: update the service deps based on NetworkScope
-	if !pushEv.full {
+	if !pushEv.pushRequest.Full {
```

Comment:
prefer to get the pushRequest first

Reply:
Do you mean check for nil or some thing else?

Reply:
Mostly to remove duplicate `pushEv.pushRequest` prefix when accessing some fields

Reply:
ok. makes sense. Changed